src/components/Icons/Plus.js

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react'
+const SvgPlus = ({ title, titleId, ...props }) => (
+  <svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" fill="none" viewBox="0 0 24 24" aria-labelledby={titleId} {...props}>
+    {title ? <title id={titleId}>{title}</title> : null}
```

Comment:
you can also write it like this:
{title && <title id={titleId}>{title}</title>}

Anton pointed it out to me yesterday🙃